<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainAirConditioningPage</class>
 <widget class="QWidget" name="MainAirConditioningPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>202</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#titleLabel {
	border: 1px solid white;
	border-radius: 10px;
	min-width: 100px;
	min-height: 20px;
}

QFrame#frame {
	border: 2px solid rgb(125, 125, 125, 50%);
	border-radius: 3px;
	background-color:rgb(100, 200, 200, 50%);
}

QPushButton {
	border-image: url(:/Hac/background-rect-button.png);
	min-height: 25px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>241</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Air Conditioning</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="4">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Floor 1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_1">
          <property name="text">
           <string>Room 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Room 2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Room 3</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Room 4</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Room 6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Room 5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Floor 2</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Room 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Room 2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Room 3</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Room 4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
